test/e2e/helpers/kafka_helper.go

```diff
@@ -252,7 +252,7 @@ func MustCreateTopic(client *testlib.Client, clusterName, clusterNamespace, topi
 	}
 }
 
-func MustCreateKafkaUserForTopic(client *testlib.Client, clusterName, clusterNamespace, userName, topicName string) {
+func MustCreateKafkaUserForTopic(client *testlib.Client, clusterName, clusterNamespace, authenticationType, userName, topicName string) {
```

Comment:
I don't see any usage of this function. Where is this used? Just curious.

Reply:
planed to be using it on `eventing-kafka-broker`.